fdbserver/EncryptKeyProxyInterface.h

```diff
@@ -155,18 +179,46 @@ struct EKPGetLatestBaseCipherKeysReply {
 	}
 };
 
+struct EKPGetLatestCipherKeysRequestInfo {
+	constexpr static FileIdentifier file_identifier = 2180516;
+	EncryptCipherDomainId domainId;
+	EncryptCipherDomainName domainName;
```

Comment:
What domain name shall be pass for system key domain and encryption header domain? Add inline comment?

Reply:
Shall we add comment to say the domain name only contain auxiliary info for key lookup, and a cipher key is still uniquely identified by the (domainId, cipherId) pair? Since the local cache of encrypt key proxy is still keyed by (domainId, cipherId) pair.

Reply:
I extended `EncryptUtils.h` to define the tenant name for non-tenant/default name, caller can override it as needed, for this change I used the default value. (`#define FDB_DEFAULT_ENCRYPT_DOMAIN_NAME "FdbDefaultEncryptDomain`)

> Add comment
Good point, I will add a comment. 
Thanks